zingocli/src/lib.rs

```diff
@@ -513,6 +514,11 @@ fn dispatch_command_or_start_interactive(cli_config: &ConfigTemplate) {
 
 /// TODO: Add Doc Comment Here!
```

Comment:
Upgrade the doc-comment, to something better.

Please add a doc-test.

Reply:
this is the entrypoint to the CLI so i'm not sure we can add a doc-test for this fn